### PR TITLE
AQICN Geoprocessor

### PR DESCRIPTION
**Pull request by mbertrand**
Scrapes the www.aqicn.org website and creates/updates a layer in GeoNode.  Only the latest data is saved to the layer, the rest is moved to an archive table.

### CODE REVIEW

File: dataqs/forecastio/tasks.py [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 
 from celery import shared_task
-from dataqs.forecastio.forecastio_air import ForecastIOAirTempProcessor
+from dataqs.aqicn.aqicn import AQICNProcessor
 
 
 @shared_task
-def forecast_io_task():
-    processor = ForecastIOAirTempProcessor()
+def aqicn_task():
```
Comment (aashish24) [inline dataqs/forecastio/tasks.py]: @mbertrand not sure if this change makes sense. What I am missing?

File: dataqs/forecastio/tasks.py [hunk 2]
```diff
@@ -1,12 +1,12 @@
 from __future__ import absolute_import
 
 from celery import shared_task
-from dataqs.forecastio.forecastio_air import ForecastIOAirTempProcessor
+from dataqs.aqicn.aqicn import AQICNProcessor
```
Comment (aashish24) [inline dataqs/forecastio/tasks.py]: @mbertrand same here. Did you change this by mistake or left over from initial debugging?

**Comment by aashish24** [comment]
@mbertrand most of the changes looks reasonable to me but some are not quite right or making sense. Please have a look at my comment. Happy New Year !! :tada:

File: dataqs/forecastio/tasks.py (hunk 1 above)
Comment (mbertrand) [inline dataqs/forecastio/tasks.py]: This was a mistake, thanks for finding it.  Just corrected it.

**Comment by aashish24** [comment]
LGTM :+1:

**Pull request merged**